airflow/logging_backends/elasticsearch_logging_backend.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from airflow.logging_backends.base_logging_backend import BaseLoggingBackend
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Search
+
+
+class ElasticsearchLoggingBackend(BaseLoggingBackend):
+
+    SCHEMA_NAME = 'elasticsearch'
+    # Maximum number of logs to return per search call.
+    MAX_PER_PAGE = 1000
+
+    def __init__(self, host=None, **kwargs):
+        super(ElasticsearchLoggingBackend, self).__init__(
+            schema_name=self.SCHEMA_NAME, **kwargs)
+
+        self.client = Elasticsearch([host])
```

Comment:
What's the error handling around Elastic search failures (e.g. if the worker didn't actually log the logs the webserver expects in elastic search)?